openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -35,6 +35,21 @@ OpenAssessment.ResponseView.prototype = {
     // Maximum file size (5 MB) for an attached file.
     MAX_FILE_SIZE: 5242880,
 
+    UPLOADABLE_IMAGE_MIME_TYPES: [
+        'image/gif',
+        'image/jpeg',
+        'image/pjpeg',
+        'image/png',
+    ],
+
+    UPLOADABLE_FILE_MIME_TYPES: [
+        'application/pdf',
+        'image/gif',
+        'image/jpeg',
+        'image/pjpeg',
+        'image/png',
+    ],
```

Comment:
Yes, but there's a tradeoff. Since this is part of an object literal, the UPLOADABLE_FILE_MIME_TYPES key would have to be set at the bottom of the file, and the two would therefore be separated by quite a distance. I think it's more readable as it is.